rsHRF_Mediated_Brain_Dynamics.ipynb

```diff
@@ -72,14 +72,14 @@
    "metadata": {},
    "source": [
     "## Hemodynamic Response Function Retrieval\n",
-    "Instead of using a constant HRF, we retrieve the same through empirical region-wise BOLD time-series data(separately for each subject, and each brain region of the subject). For more information, visit: https://www.nitrc.org/projects/rshrf.\n",
+    "Instead of using a constant HRF, we retrieve the same through empirical region-wise BOLD time-series data (separately for each subject, and each brain region of the subject). For more information, visit: https://www.nitrc.org/projects/rshrf.\n",
     "\n",
     "<hr style=\"border:2px solid gray\"> </hr>\n",
     "\n",
     "![Resting-State HRF Retrieval](https://guorongwu.github.io/HRF/deconv.jpg)\n",
     "\n",
     "<hr style=\"border:2px solid gray\"> </hr>\n",
-    "Here, we carry out the steps 1 and 2 as shown in the figure. The region-wise emperical time-series forms the input and the HRF is retrieved from the same. The consequent BOLD simulations are then mediated through the retrieved HRF."
+    "Here, we carry out the steps 1 and 2 as shown in the figure. The region-wise empirical time-series forms the input and the HRF is retrieved from the same. The consequent neural simulations are then converted to BOLD through the retrieved HRF and downsampling"
```

Comment:
In the python-version (and even the final C version) we are not explicitly down-sampling. Instead we are obtaining convolution-sum (BOLD response) for only those a time-steps which coincide with BOLD Repetition-Time (the formal proof for this is [here](https://github.com/danielemarinazzo/TVB_convolution_HRF/blob/amogh/Proof_Of_Concept.md).) Hence, I feel mentioning down-sampling might give the idea that we are adding another function in the existing TVB workflow.